5.5/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openshift/base-centos7
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+MAINTAINER Ben Parees <bparees@redhat.com>
+
+# Image metadata
+ENV PHP_VERSION           5.5
+ENV IMAGE_DESCRIPTION     PHP 5.5
+ENV IMAGE_TAGS            php,php55
+ENV IMAGE_EXPOSE_SERVICES 8080:http
+#ENV PHPRC /opt/openshift/etc/php.ini
+#ENV PHP_INI_SCAN_DIR /opt/openshift/etc/php.d
```

Comment:
I think the intention here was to show that you can hardcode them into Docker image, which is not a bad idea if documented (if you use the layered approach).